thirdeye/thirdeye-pinot/src/main/java/com/linkedin/thirdeye/anomaly/merge/AnomalyMergeExecutor.java

```diff
@@ -270,6 +278,29 @@ private void updateMergedAnomalyWeight(MergedAnomalyResultDTO anomalyMergedResul
         }
       }
 
+      // Transform Time Series
```

Comment:
Moved the duplicate code to this method "List<ScalingFactor> getTimeSeriesScalingFactors" of OverrideConfigHelper.